actors/uploader/main.js

```diff
@@ -10,6 +10,7 @@ import gzip from "node-gzip";
 import { DynamoDBClient, PutItemCommand } from "@aws-sdk/client-dynamodb";
 
 import { keboolaUploader } from "./src/uploader.js";
+import { SmartyValidator } from "./src/validators/smartyValidator.js";
```

Comment:
Validatory prosím nepřidávat, actor by měl vracet data v shape, který je ready na upload a nemá se to hackovat až v týhloe fázi, cílem je se validátorů zbavit úplně, ne přidávat další

Reply:
happily smazano, pridal jsem jen proto ze je meli vsechny ostatni